ros2/naive/CMakeLists.txt

```
@@ -0,0 +1,34 @@
+cmake_minimum_required(VERSION 3.8)
+project(naive)
```

Specify the languages you're using. If you're only using C++, then that means doing the following
```suggestion
project(naive CXX)
```

---

I see. Changed.